invenio/templates/configurations/logstash.yaml

```diff
@@ -53,66 +53,125 @@ data:
         # If request is a ping, then ignore it.
 
         if [data][url] != "ping" {
+          drop{}
+        }
 
```

Comment:
Changes from here to line 110 are just indentation changes, since the `pings`are dropped in the previous line